spec/game_spec.rb

```diff
@@ -88,18 +88,48 @@
     GAME
   end
 
+  it "prints game over when the player wins" do
+    stdin = StringIO.new
+    stdout = StringIO.new
+    grid = Grid.new(
```

Comment:
I like the way you are lining up the variable so that you can see how they won or lost.